web/src/main/webapp/my-app/marketplace/partials/marketplace.html

```diff
@@ -18,61 +18,70 @@
     under the License.
 
 -->
-<frame-page app-title="Browse {{NAMES.title}}" app-icon="home" app-show-add-to-home="false">
+<frame-page>
```

Comment:
I think we'll probably still need `app-title` to inform accessible page titles.

Reply:
"App title" is still supported in the `<frame-page>` directive. In this case it was replaced with a different `<h1>` tag containing the same thing that would otherwise be in the `app-title` attribute. I went this route for the sake of a consistent appearance among this app's pages. Without `white-background` on this page, the markup generated by `app-title` looks a little too different from the rest of the app's pages (which have `white-background` set to true).